src/Grid.ts

```diff
@@ -1,22 +1,21 @@
 import { Subscription } from '@dojo/shim/Observable';
 import { v, w } from '@dojo/widget-core/d';
-import { DNode, PropertyChangeRecord } from '@dojo/widget-core/interfaces';
+import { DNode, PropertyChangeRecord, WidgetBaseConstructor } from '@dojo/widget-core/interfaces';
 import { RegistryMixin }  from '@dojo/widget-core/mixins/Registry';
 import { theme, ThemeableMixin, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
 import WidgetBase, { diffProperty } from '@dojo/widget-core/WidgetBase';
-import WidgetRegistry from '@dojo/widget-core/WidgetRegistry';
 import DataProviderBase from './bases/DataProviderBase';
 import Body from './Body';
-import Cell from './Cell';
+import GridRegistry, { gridRegistry } from './GridRegistry';
```

Comment:
I'm actually using `registry` as the name of the property the user can pass and gridRegistry is the default to use if they don't